cmapi/cmapi_server/constants.py

```diff
@@ -82,3 +82,8 @@ class ProgInfo(NamedTuple):
 IFLAG = os.path.join(MCS_ETC_PATH, 'container-initialized')
 LIBJEMALLOC_DEFAULT_PATH = os.path.join(MCS_DATA_PATH, 'libjemalloc.so.2')
 MCS_LOG_PATH = '/var/log/mariadb/columnstore'
+
+
+# client constants
+REQUEST_TIMEOUT: float = 330.0  # 5,5 minutes
```

Comment:
We need to experiment but to me these values are method-specific, e.g. `shutdown_cluster` should have it equal to graceful_timeout, `start_cluster` can have it as 330 secs,  but `set_mode`, `get_status`, should have lower values, so if it fails, it fails faster(30 secs should be enough). `load_s3` on contrary must have huge value here.